### ISSUE TITLE
Re-configure descriptions of FlowJo templates

### ISSUE THREAD

**Issue by martin1english** [tercen/flowjo_plugin#61]
![image](https://user-images.githubusercontent.com/64135638/146229605-281364c0-2696-4b81-9137-fb165b5c946a.png)

@MahoNagasawa @fnaji 
I would like to do some more work on these titles and descriptions because I feel they are too generic. 

I think we have an opportunity to give good information here and prevent confusion. 

The titles and descriptions should clearly tell the user the analysis they will get when they choose that template

For example something like this.

Original
Title - flowsom flowjo 0.0.1
Description - A simple workflow for flowjo flowSOM n=15

Change to?
Title - FlowSOM & Cluster Heatmaps 1.0.1
Description - Run flowSOM with n=15 and view clusters in a heatmap format.

**Comment by martin1english** [comment]
@MahoNagasawa You can use special characters in the descriptions but not in the Titles.

Would you put your suggestions for changes here. I don't think my new ones above are quite right. You might need to refine them.

**Comment by MahoNagasawa** [comment]
@martin1english @fnaji 
How about those.

template 1:
title: Tercen plugin FlowSOM
description: For data set uploaded from FlowJo with Tercen plugin. Runs FlowSOM with cluster number 15. Views clusters in a heatmap format. 

template 2:
title: Tercen plugin FlowSOM tSNE UMAP
description: For data set uploaded from FlowJo with Tercen plugin. Runs FlowSOM with cluster number 15. Views FlowSOM clusters in a heatmap format, tSNE and UMAP clusters in a 2D plot format.

**Comment by martin1english** [comment]
Ideas for names:

Change to
Title - Cluster Exploration 0.0.1
Title - FlowJo Cluster Exploration 0.0.1
Title - Population Exploration 0.0.1
Title - FlowSOM with Cluster Heatmaps 0.0.1
Description - For use with FlowJo. Run flowSOM with n=15 and view clusters in a heatmap format.


Change to
Title - FlowSOM with Cluster Heatmap & tSNE & UMAP 1.0.0
description - Run flowSOM with n=15 and view clusters in a heatmap format, RUN tSNE and UMAP and view in a 2D plot

**Comment by MahoNagasawa** [comment]
Since I prepared the new templates with random label step and applied less than 1 filter on FlowSOM, I gave the following titles and description for the time being. (it's still an open question for the title and description.)

Title1: Cluster explorer basic FJ
Description1: Workflow template for FlowJo Tercen plugin uploaded files. FlowSOM  (cluster n=15)  view in a heatmap. Downsampling applied (1%) 

Title2: Cluster explorer advanced FJ
Description 2: Workflow template for FlowJo Tercen plugin uploaded files. FlowSOM  (cluster n=15)  view in a heatmap and MST. 2D plot with tSNE and UMAP . Downsampling applied (1%)
